### PR TITLE
Split up line 204 which was greater than 72 characters

### PR DESCRIPTION
**Pull request by asoloway64**

### CODE REVIEW

File: draft-ietf-core-conditional-attributes.md [hunk 1]
```diff
@@ -201,7 +201,9 @@ bool notifiable( Resource * r ) {
 #define LT_EX ( r->v < r->lt ^ r->last_rep_v < r->lt )
 #define GT_EX ( r->v > r->gt ^ r->last_rep_v > r->gt )
 #define ST_EX ( abs( r->v - r->last_rep_v ) >= r->st )
-#define IN_BAND ( ( r->gt <= r->v && r->v <= r->lt ) || ( r->lt <= r->gt && r->gt <= r->v ) || ( r->v <= r->lt && r->lt <= r->gt ) )
+#define IN_BAND ( ( r->gt <= r->v && r->v <= r->lt ) ||
+                  ( r->lt <= r->gt && r->gt <= r->v ) ||
```
Comment (cabo) [inline draft-ietf-core-conditional-attributes.md:205]: C requires backslash-escaping continuation lines of #defines...

**Review by bsilverajan — CHANGES_REQUESTED** [review]
Thanks Alan and Carsten. 

Alan could you please amend the change to include ~~leading backslashes to lines 205 and 206~~  trailing backslashes to lines 204 and 205?

(Edited based on Carsten's feedback)

**Comment by cabo** [comment]
Actually, I think you need trailing backslashes on 204 and 205, but I haven't used continuation lines on #define for quite some time.

File: draft-ietf-core-conditional-attributes.md (hunk 1 above)
Comment (bsilverajan) [reply draft-ietf-core-conditional-attributes.md:205]: Very true, that was my mistake!

**Comment by bsilverajan** [comment]
> Actually, I think you need trailing backslashes on 204 and 205, but I haven't used continuation lines on #define for quite some time.

Very true, that was my mistake!

**Comment by asoloway64** [comment]
Markdown uses backslash for control characters, so my understanding was that you needed the double-backslash to get a backslash displayed.

**Comment by bsilverajan** [comment]
> Markdown uses backslash for control characters, so my understanding was that you needed the double-backslash to get a backslash displayed.

I ran both versions through xml2rfc and it seems just a single backslash suffices. So I think we can merge the PR with Carsten's changes. 

Thank you both!

**Pull request merged**